src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTriggerTest.java

```diff
@@ -2100,12 +2091,6 @@ public void dependencySetUp() {
         when(downstreamProject.getFullName()).thenReturn("MockedProject");
         when(upstreamProject.getFullName()).thenReturn("MockedUpstreamProject");
         when(veryUpstreamProject.getFullName()).thenReturn("MockedVeryUpstreamProject");
-        when(hudsonMock.getItem(eq("MockedProject"), any(Item.class), eq(Item.class))).
```

Comment:
?

Reply:
Did you have a question, Robert?

Reply:
Yes, I was wondering why this got removed?

Reply:
Because it's no longer needed.